### PR TITLE
Fixes for FLD FN Issues

### PR DESCRIPTION
**Pull request by 0xtaf**
related issue: https://github.com/forta-network/starter-kits/issues/427 https://github.com/forta-network/starter-kits/issues/428

- currency symbol for avax has changed
- blocks in memory values updated
- railgun addresses are added
- mixer labels are fixed
- timestamp checker for first_tx calculation is added
- token price fetcher is fixed
- arbitrum bridge checker is added

### CODE REVIEW

File: funding-laundering-detector-py/src/agent.py [hunk 1]
```diff
@@ -268,13 +309,17 @@ async def analyze_transaction(transaction_event: forta_agent.transaction_event.T
...
-        elif to in confirmed_targets_keys and from_ not in confirmed_targets_keys:
+        elif to in confirmed_targets_keys and from_ not in confirmed_targets_keys or to in mixer_addresses.get(CHAIN_ID, []):
             DENOMINATOR_COUNT_LAUNDERING += 1
+            if to in mixer_addresses.get(CHAIN_ID, []):
+                label = "mixer"
+            else:
+                label = confirmed_targets[to]['type']
             usd, token = calculate_usd_and_get_symbol(web3, event.address.lower(), ERC20_ABI, value)
             if usd > thresholds["TRANSFER_THRESHOLD_IN_USD"] and (
                     confirmed_targets[to]['type'] != 'dex' or not DEX_DISABLE) and (
```
Comment (Vxatz) [inline funding-laundering-detector-py/src/agent.py:320]:
```suggestion
            if usd > thresholds["TRANSFER_THRESHOLD_IN_USD"] and (
                    (to in confirmed_targets and confirmed_targets[to]['type'] != 'dex') or not DEX_DISABLE) and (
```

File: funding-laundering-detector-py/src/agent.py [hunk 2]
```diff
@@ -345,13 +390,17 @@ async def analyze_transaction(transaction_event: forta_agent.transaction_event.T
...
-        elif to in confirmed_targets_keys and from_ not in confirmed_targets_keys:
+        elif to in confirmed_targets_keys and from_ not in confirmed_targets_keys or to in mixer_addresses.get(CHAIN_ID, []):
             DENOMINATOR_COUNT_LAUNDERING += 1
+            if to in mixer_addresses.get(CHAIN_ID, []):
+                label = "mixer"
+            else:
+                label = confirmed_targets[to]['type']
             usd, token = calculate_usd_for_base_token(value, CHAIN_ID)
             if usd > thresholds["TRANSFER_THRESHOLD_IN_USD"] and (
                     confirmed_targets[to]['type'] != 'dex' or not DEX_DISABLE) and (
```
Comment (Vxatz) [inline funding-laundering-detector-py/src/agent.py:401]:
```suggestion
            if usd > thresholds["TRANSFER_THRESHOLD_IN_USD"] and (
                    (to in confirmed_targets and confirmed_targets[to]['type'] != 'dex') or not DEX_DISABLE) and (
```

**Comment by Vxatz** [comment]
Passing `confirmed_targets[to]['type']`
should also be updated to `confirmed_targets[to]['type'] if to in confirmed_targets else "mixer"` when appending the laundering findings [here](https://github.com/forta-network/starter-kits/blob/a52206df81132d4f717140e09620f2348f79d641/funding-laundering-detector-py/src/agent.py#L341) and [here](https://github.com/forta-network/starter-kits/blob/a52206df81132d4f717140e09620f2348f79d641/funding-laundering-detector-py/src/agent.py#L423)

**Comment by Vxatz** [comment]
Can add one more "9" [here](https://github.com/forta-network/starter-kits/blob/a52206df81132d4f717140e09620f2348f79d641/funding-laundering-detector-py/src/blockexplorer.py#L43) as the block number has been surpassed on Arbitrum

File: funding-laundering-detector-py/src/agent.py (hunk 2 above)
Comment (0xtaf) [reply funding-laundering-detector-py/src/agent.py:401]: `to in confirmed_targets` will not be met if it's just manually caught from the mixers 🤔

**Comment by 0xtaf** [comment]
> Passing `confirmed_targets[to]['type']` should also be updated to `confirmed_targets[to]['type'] if to in confirmed_targets else "mixer"` when appending the laundering findings [here](https://github.com/forta-network/starter-kits/blob/a52206df81132d4f717140e09620f2348f79d641/funding-laundering-detector-py/src/agent.py#L341) and [here](https://github.com/forta-network/starter-kits/blob/a52206df81132d4f717140e09620f2348f79d641/funding-laundering-detector-py/src/agent.py#L423)

or I can just pass the label because of this block 

```
            if to in mixer_addresses.get(CHAIN_ID, []):
                label = "mixer"
            else:
                label = confirmed_targets[to]['type']
```

**Review by Vxatz — APPROVED** [review]

**Pull request merged**